usage/app/model/ContentDetails.scala

```diff
@@ -20,7 +24,15 @@ object ContentDetails {
     ContentDetails(
       content.webTitle,
       content.webUrl,
-      content.sectionId.getOrElse("none")
+      content.sectionId.getOrElse("none"),
+      composerUrl(content)
     )
   }
+
+  def composerUrl(content: Content): Option[URL] = content.fields
+    .flatMap(_.internalComposerCode)
+    .flatMap(composerId => {
+      Try(new URL(s"${Config.composerBaseUrl}/${composerId}")).toOption
```

Comment:
In the future it could be nice to denormalise this into the usage metadata at collection time, rather than having to construct this here, which will imply knowledge of all the systems this integrates with.